src/qibo/models/dbi/utils.py

```diff
@@ -163,3 +151,27 @@ def cs_angle_sgn(dbi_object, d):
         )
     )
     return np.sign(norm)
+
+
+def off_diagonal_norm_polynomial_expansion_coef(dbi_object, d, n):
```

Comment:
@Sam-XiaoyueLi  says this is also used for the $\partial_{B_i}(f_{loss})$ so it's not just for scheduling. Maybe worth discussing some more